config.make

```diff
@@ -128,18 +128,38 @@ ZFP_INC = $(ZFP_HOME)/inc
 else
 ZFP_INC = $(ZFP_HOME)/include
 endif
+ifeq ($(wildcard $(ZFP_HOME)/lib),)
```

Comment:
What is `wildcard`? I've never seen that before.

Reply:
https://www.gnu.org/software/make/manual/html_node/Wildcard-Function.html